### ISSUE TITLE
Get CoopHive dev environment working on Mac

### ISSUE THREAD

**Issue by lukemarsden** [CoopHive/coophive#9]
In a PR, update https://github.com/CoopHive/coophive/blob/main/ARCHITECTURE.md#local-development with alternative instructions for setting things up on a Mac

- [ ] add alternative setup instructions to pre-requisites section
- [ ] we currently use a bacalhau fork https://github.com/CoopHive/bacalhau which only has a released binary for linux x86_64, need to release a binary for darwin arm64 (build in docker? https://github.com/CoopHive/bacalhau/blob/v1.0.3-coophive-hacks/build.sh or failing that, downgrade go version to 1.20 or similar?)
- [ ] run through rest of steps and see what breaks and then fix it

**Comment by gpylypchuk** [comment]
@mactus13 I think you could use ```docker pull ghcr.io/bacalhau-project/bacalhau:v1.0.3``` and then run it ```docker run -it ghcr.io/bacalhau-project/bacalhau:v1.0.3``` then you should be able to 
```bash
docker run -t ghcr.io/bacalhau-project/bacalhau:v1.0.3 \
                docker run \
                --id-only \
                --wait \
                ubuntu:latest -- \
                sh -c 'uname -a && echo "Hello from Docker Bacalhau!"'```

**Comment by lukemarsden** [comment]
I was thinking more of just running this to build on Mac

https://github.com/CoopHive/bacalhau/blob/v1.0.3-coophive-hacks/build.sh


lukemarsden.net
***@***.***
@lmarsden <https://twitter.com/lmarsden>

**Comment by gpylypchuk** [comment]
Yes, but the problem also was the golang packages if that goes well then should work, @mactus13, could you install that?

**Comment by mactus13** [comment]
Pair with @lukemarsden and found out that the docker container that runs Geth is broken and it is not working thus causing the balances errors.
.
Also Geth version 1.14 still has a problem as it doesn't recognise eth.coinbase so we had to use 1.13.5.
.
We've tested ./stack fund-admin that uses local installed Geth and it worked so we settle up upon modifying the stack script to have a new argument that will be used in development environments only and will point to a local Geth and all the current command won't be modified so nothing we'll be affected in production environment which runs on (AMD/Intel) arch.
